pandas/io/stata.py

```diff
@@ -160,15 +160,26 @@
 
 Examples
 --------
+
+Creating a dummy stata for this example
+>>>df = pd.DataFrame({'animal': ['falcon', 'parrot', 'falcon',
+...                              'parrot'],
+...                   'speed': [350, 18, 361, 15]})
+>>>df.to_stata('animals.dta')
 Read a Stata dta file:
 
->>> df = pd.read_stata('filename.dta')
+>>> df = pd.read_stata('animals.dta')
 
 Read a Stata dta file in 10,000 line chunks:
-
+>>> filename_df = pd.DataFrame(np.random.randint(0,10, size=(10,000,1)))
```

Comment:
size=(10,000,1) is not a legal Python expression.  You should go with

I would use

```
values = np.random.randint(0,10, size=(20_000,1), dtype="uint8")
df = pd.DataFrame(values, columns=["i"])
df.to_stata('filename.dta')
```

```
itr = pd.read_stata('filename.dta', chunksize=10000)
for chunk in itr:
    # Operate on a single chunk
    chunk.mean()
```

1. The type will ensure the file is always small. 
2. Setting the column will prevent a warning
3. 20k records means the iteration is non-trivial